refreshengine.go

```
@@ -101,41 +112,23 @@ func (pd *PopData) RefreshEngine(conf *Config, stopch chan struct{}) {
+					pd.ComponentStatusCh <- tapir.ComponentStatusUpdate{
 						Status:    tapir.StatusFail,
 						Component: "tapir-observation",
 						Msg:       fmt.Sprintf("ProcessTapirUpdate error: %v", err),
 					}
 					log.Printf("RefreshEngine: Error from ProcessTapirUpdate(): %v", err)
 				}
-				Gconfig.Internal.ComponentStatusCh <- tapir.ComponentStatusUpdate{
+				pd.ComponentStatusCh <- tapir.ComponentStatusUpdate{
 					Status:    tapir.StatusOK,
 					Component: "tapir-observation",
 					Msg:       fmt.Sprintf("ProcessTapirUpdate: MQTT observation message received"),
```

golangci-lint (2.12.1)

[error] S1039: unnecessary use of fmt.Sprintf

(staticcheck)